test/articledownloadlinkslist.spec.js

```diff
@@ -11,10 +17,39 @@ describe('An ArticleDownloadLinksList Component', function () {
 
   beforeEach(function () {
     articleDownloadLinksList = new ArticleDownloadLinksList($el, window, window.document);
+    spy(articleDownloadLinksList.$elm, 'setAttribute');
+  });
+
+  afterEach(function() {
+    articleDownloadLinksList.$elm.setAttribute.restore();
+  });
+
+  describe('The open method', function () {
+
+    it('Sets aria-expanded attribute to "true"', function () {
+      articleDownloadLinksList.open();
+      expect(articleDownloadLinksList.$elm.setAttribute.calledWithExactly('aria-expanded', true)).to.be.true;
+    })
+
+    it('Sets class "visuallyhidden" to be removed', function () {
```

Comment:
`'removes the class "visuallyhidden"'` reads better.

Reply:
Completed